wiki/Gameplay/de.md

```diff
@@ -0,0 +1,13 @@
+# Gameplay
+
+- [Genauigkeit](/wiki/Gameplay/Accuracy)
+- [Drain-Zeit](/wiki/Gameplay/Drain_time)
+- [Fingerlock](/wiki/Gameplay/Fingerlock)
+- [Eingabegerät](/wiki/Gameplay/Input_device)
+- [Mind block](/wiki/Gameplay/Mind_block)
```

Comment:
My suggestion is `Denkblockade` but I'm not so sure myself since `Mindblock` is used a bit more often nowadays in German. I will let you take the decision.

Reply:
`Denkblockade` sounds good